vtysh/bgp_vty.c

```diff
@@ -3398,6 +3398,8 @@ cli_neighbor_set_peer_group_cmd_execute(char *vrf_name, const char *ip_addr,
     const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor;
     const struct ovsrec_bgp_neighbor *ovs_bgp_peer_group;
     struct ovsdb_idl_txn *txn;
+    char *key_timers[BGP_MAX_TIMERS];
```

Comment:
in the source code we assume that the array size is 2. So, you can easily use magic 2 instead. If you do not use magic 2, then do not use everywhere and not only in that line.